plutus-pab/test-node/testnet/pab-config.yml

```
@@ -46,6 +46,9 @@ signingProcessConfig:
 metadataServerConfig:
   mdBaseUrl: http://localhost:9085
 
+developmentOptions:
+  pabRollbackHistory: null
+
```

It should be fine. However we need to add some more things.
```suggestion
developmentOptions:
  pabRollbackHistory: null
  pabResumeFrom:
    { "tag" : "PointAtGenesis" }
    # { "tag" : "Point"
    # , "pointBlockId" : "613d23b8df670e3692f4e4ab59b179778519de15999ee1a51077fcc02a48abaa"
    # , "pointSlot" : { "getSlot" : 45656608 }
    # }

```